services/queue/src/data.js

```diff
@@ -168,6 +164,8 @@ class Task {
         };
       }),
     };
+    addSplitFields(status);
```

Comment:
I see that `definition()` doesn't add the split fields, while `status()` does.  It'd be good to be consistent, either way.  I have a mild preference for both functions returning `taskQueueId` and isolating all of the splitting into `api.js`.

Reply:
Yeah, that's exactly what my earlier question about "seams" was referring to. It's just that I found `status()` was called in more places than just `api.js` which was what led me to treat it differently. I think I agree with your preference here despite meaning more code overall.